### ISSUE TITLE
Node.js Foundation Core Technical Committee (CTC) Meeting 2017-03-08

### ISSUE THREAD

**Issue by Trott** [nodejs/CTC#77]
## Time

**UTC Wed 08-Mar-2017 16:00 (04:00 PM)**:

| Timezone      | Date/Time             |
|---------------|-----------------------|
| US / Pacific  | Wed 08-Mar-2017 08:00 (08:00 AM) |
| US / Mountain | Wed 08-Mar-2017 09:00 (09:00 AM) |
| US / Central  | Wed 08-Mar-2017 10:00 (10:00 AM) |
| US / Eastern  | Wed 08-Mar-2017 11:00 (11:00 AM) |
| Amsterdam     | Wed 08-Mar-2017 17:00 (05:00 PM) |
| Moscow        | Wed 08-Mar-2017 19:00 (07:00 PM) |
| Chennai       | Wed 08-Mar-2017 21:30 (09:30 PM) |
| Tokyo         | Thu 09-Mar-2017 01:00 (01:00 AM) |
| Sydney        | Thu 09-Mar-2017 03:00 (03:00 AM) |

Or in your local time:
* http://www.timeanddate.com/worldclock/fixedtime.html?msg=Node.js+Foundation+Core%20Technical%20Committee%20(CTC)+Meeting+2017-03-08&iso=20170308T16
* or http://www.wolframalpha.com/input/?i=4pm+UTC%2C+Mar+08%2C+2017+in+local+time

## Links

* **Minutes Google Doc**: <https://docs.google.com/document/d/1yTsNRUQ8hO0LvaquRx4TDsJMakDxKfVIRidViemZFmQ>
* _Previous Minutes Google Doc: <https://docs.google.com/document/d/1xoSIVTwNoOfOf0NSajbZudD2HX4PvYQ5V9eVqfDu0qs>_

## Agenda

Extracted from **ctc-agenda** labelled issues and pull requests from the **nodejs org** prior to the meeting.

### nodejs/node

* governance: add Franziska Hinkelmann to the CTC [#11488](https://github.com/nodejs/node/pull/11488) @rvagg
* Symbol key props visible in inspection by default [#9726](https://github.com/nodejs/node/pull/9726) @Trott

### nodejs/TSC

* Updating the Copyright [#174](https://github.com/nodejs/TSC/issues/174) @jasnell

### nodejs/node-eps

* 002 - web compat of file extension/index searching [#51](https://github.com/nodejs/node-eps/issues/51) @bmeck

## Invited

* Anna Henningsen @addaleax (CTC)
* Bradley Meck @bmeck (observer/GoDaddy/TC39)
* Ben Noordhuis @bnoordhuis (CTC)
* Сковорода Никита Андреевич @ChALkeR (CTC)
* Colin Ihrig @cjihrig (CTC)
* Evan Lucas @evanlucas (CTC)
* Franziska Hinkelmann @fhinkel (observer/Google)
* Jeremiah Senkpiel @Fishrock123 (CTC)
* Tracy Hinds @hackygolucky (observer/Node.js Foundation)
* Fedor Indutny @indutny (CTC)
* James M Snell @jasnell (CTC)
* Josh Gavant @joshgav (observer/Microsoft)
* Michael Dawson @mhdawson (CTC)
* Julien Gilli @misterdjules (CTC)
* Mikeal Rogers @mikeal (observer/Node.js Foundation)
* Brian White @mscdex (CTC)
* Myles Borins @MylesBorins (CTC)
* Ali Ijaz Sheikh @ofrobots (CTC)
* Jenn Turner @renrutnnej (observer/Node.js Foundation)
* Rod Vagg @rvagg (CTC)
* Seth Thompson @s3ththompson (observer/Google)
* Shigeki Ohtsu @shigeki (CTC)
* Steven R Loomis @srl295 (observer/IBM/ICU)
* Michaël Zasso @targos (CTC)
* Sakthipriyan Vairamani @thefourtheye (CTC)
* Trevor Norris @trevnorris (CTC)
* Rich Trott @Trott (CTC)

## Notes

The agenda comes from issues labelled with `ctc-agenda` across **all of the repositories in the nodejs org**. Please label any additional issues that should be on the agenda before the meeting starts.

## Joining the meeting

Uberconference; participants should have the link & numbers, contact me if you don't.

## Public participation

We stream our conference call straight to YouTube so anyone can listen to it live, it should start playing at **https://www.youtube.com/c/nodejs+foundation/live** when we turn it on. There's usually a short cat-herding time at the start of the meeting and then occasionally we have some quick private business to attend to before we can start recording & streaming. So be patient and it should show up.

Many of us will be on IRC in #node-dev on Freenode if you'd like to interact, we have a Q/A session scheduled at the end of the meeting if you'd like us to discuss anything in particular. @nodejs/collaborators in particular if there's anything you need from the CTC that's not worth putting on as a separate agenda item, this is a good place for it.

**Comment by Trott** [comment]
Items labeled `ctc-review`. Not on the agenda this week but many are definitely worth your attention:

* [buffer: discuss future direction of Buffer constructor API](https://github.com/nodejs/node/issues/9531)
* [build: remove cares header from tarball](https://github.com/nodejs/node/pull/10283)
* [tty: remove NODE_TTY_UNSAFE_ASYNC](https://github.com/nodejs/node/pull/10393)
* [module: improve require() performance](https://github.com/nodejs/node/pull/10789)
* [http: add deprecation warnings for private props](https://github.com/nodejs/node/pull/10941)
* [http: append Cookie header values with semicolon](https://github.com/nodejs/node/pull/11259)
* [tls: runtime deprecate _tls_legacy](https://github.com/nodejs/node/pull/11349)
* [process: implement resourceUsage()](https://github.com/nodejs/node/pull/11490)
* [repl: docs-only deprecate magic mode](https://github.com/nodejs/node/pull/11599)

**Comment by Trott** [comment]
@nodejs/ctc Items labeled `ctc-review`. Not on the agenda this week but many are definitely worth your attention:

* [buffer: discuss future direction of Buffer constructor API](https://github.com/nodejs/node/issues/9531)
* [build: remove cares header from tarball](https://github.com/nodejs/node/pull/10283)
* [tty: remove NODE_TTY_UNSAFE_ASYNC](https://github.com/nodejs/node/pull/10393)
* [module: improve require() performance](https://github.com/nodejs/node/pull/10789)
* [http: add deprecation warnings for private props](https://github.com/nodejs/node/pull/10941)
* [http: append Cookie header values with semicolon](https://github.com/nodejs/node/pull/11259)
* ~~[tls: runtime deprecate _tls_legacy](https://github.com/nodejs/node/pull/11349)~~ landed
* [process: implement resourceUsage()](https://github.com/nodejs/node/pull/11490)
* [repl: docs-only deprecate magic mode](https://github.com/nodejs/node/pull/11599)

**Comment by Trott** [comment]
I'd especially like to call people's attention to [Symbol key props visible in inspection by default](https://github.com/nodejs/node/pull/9726) as the intention there is to call for a vote. There is an impasse after lengthy conversation. Hopefully, as many of @nodejs/ctc as possible will read over the issue, and we can vote to either accept or decline the PR.

**Comment by jasnell** [comment]
I'll have an update on Friday's VM summit in Seattle.

**Comment by Trott** [comment]
@nodejs/ctc For the two items that are exceedingly likely to be brought to vote, I've added counts in the Google doc. However, I'm doing some interpretation, so please double check my work and make a change if I've messed up your vote.

**Comment by Trott** [comment]
@bmeck Regarding [002 - web compat of file extension/index searching #51](https://github.com/nodejs/node-eps/issues/51): Are you looking to CTC to make a decision? Or is it mostly that CTC should be aware of the issue and the conversation?

**Comment by addaleax** [comment]
Sorry, probably can’t attend this one, too.

**Comment by Trott** [comment]
I'll probably miss the first half hour.

**Comment by cjihrig** [comment]
I won't be able to make it this week.

**Comment by joshgav** [comment]
Can we decide if Node 8 will start with V8 5.7? See [node#10117](https://github.com/nodejs/node/issues/10117), specifically [this comment](https://github.com/nodejs/node/issues/10117#issuecomment-264954525). This decision is important for tools which work with the legacy debugger.

**Comment by misterdjules** [comment]
I won't be able to make it this time.

**Comment by bnoordhuis** [comment]
@joshgav Is there a separate issue I can tag ctc-agenda?

**Comment by joshgav** [comment]
@bnoordhuis would it be [#10117](https://github.com/nodejs/node/issues/10117)?

**Comment by bnoordhuis** [comment]
I was hoping for one that was specifically about the debugger.  We'll just have to remember to bring it up tomorrow.

**Comment by ChALkeR** [comment]
I would also ask everyone to take a look in advance (if you have several minutes) at the email and the google spreadsheet from me that you should have received, that will make discussing it easier.

**Comment by targos** [comment]
I won't be able to make it.

**Comment by bnoordhuis** [comment]
I'm having connection problems, I keep getting dropped from the call.

**Comment by joshgav** [comment]
Notes in https://github.com/nodejs/CTC/pull/78
